src/language-js/printer-estree.js

```diff
@@ -1080,12 +1081,14 @@ function printPathNoParens(path, options, print, args) {
       } else if (
         (n.importKind && n.importKind === "type") ||
         // import {} from 'x'
-        /{\s*}/.test(
+        stripComments(
```

Comment:
Is it really worth adding a new dependency just for this?

Reply:
Agree. Prettier has its own utils for this (`getNextNonSpaceNonCommentCharacterIndexWithStartIndex`).

Reply:
Good  to  know, I'll try.